README.md

```diff
@@ -14,6 +14,25 @@ The connector supports:
 - batching;
 - delivery retries.
 
+An example of configuration :  
```

Comment:
Linter is failing here, please change to (remove spaces in the end of the line):

```suggestion
An example of configuration:
```